app/build.gradle

```diff
@@ -14,8 +14,8 @@ android {
         applicationId "org.openobservatory.ooniprobe"
         minSdkVersion 14
         targetSdkVersion 25
-        versionCode 11
-        versionName "1.1.5"
+        versionCode 12
+        versionName "1.2.0"
```

Comment:
Here on Android, where you can do whatever you want with `versionName`, I suggest to add the `-rc` suffix and only leave it when we're about to release.